Vision/Vision.bs

```diff
@@ -130,6 +130,8 @@ Markup Shorthands: markdown yes
 	<li id=diversity>
 		**Diversity**: We believe in diversity
 		and inclusion of participants from different
```

Comment:
```suggestion
		and inclusion of participants of different
```

I suggest rephrasing to "of different ages", as "from different ages" means something quite different.